docs/conf.py

```diff
@@ -356,4 +356,13 @@
     ("py:mod", "aiohttp"),  # undocumented, no `.. currentmodule:: aiohttp` in docs
     ("py:class", "aiohttp.SimpleCookie"),  # undocumented
     ("py:class", "aiohttp.web.RequestHandler"),  # undocumented
+    ("py:meth", "aiohttp.ClientSession.request"),  # undocumented
+    ("py:class", "aiohttp.protocol.HttpVersion"),  # undocumented
+    ("py:class", "aiohttp.ClientRequest"),  # undocumented
+    ("py:class", "Payload"),  # undocumented
+    ("py:class", "aiohttp.abc.AbstractResolver"),  # undocumented
+    ("py:func", "aiohttp.ws_connect"),  # undocumented
+    ("py:meth", "start"),  # undocumented
```

Comment:
Same here: it should have the full context.

Reply:
`aiohttp.start.start`?

Reply:
No, the thing this refers to should actually exist in the code.

Reply:
Hmm, It will be quite hard for me, but I try

Reply:
Find what it refers to in the context where it's used and that should be it

Reply:
Opposite :meth:(close) refers to `aiohttp.WSCloseCode.OK`. Not enough context to define `start` for me

Reply:
Alright, I did some research and it is indeed unobvious what this refers to. So let's leave it be for now.